src/Resources/contao/classes/SurveyQuestionMultiplechoice.php

```diff
@@ -308,7 +306,7 @@ protected function calculateCumulated(): void
                 }
             } else {
                 if (!empty($arrAnswer['value'])) {
-                    ++$cumulated[$arrAnswer['value']];
+                    $cumulated[$arrAnswer['value']] = ($cumulated[$arrAnswer['value']] ?? 0) + 1;
```

Comment:
Does this really work? My opinion is the following: If (under PHP 8+) a key (here $arrAnswer['value']) of an array (here $cumulated) is invalid, the expression $cumulated[$arrAnswer['value']] will never evaluate correctly to NULL. Thus, the right-hand side expression cannot be processed correctly under all circumstances and a warning may occur.

Reply:
In line 308 we make an empty check on $arrAnswer['value'], so there will never be an value where ?? would get an invalid value. But ?? will not check $arrAnswer['value'], only $comulated[...] in this line I think?

Reply:
yes, of course - you are right - I am only looking at 309 - not 308 :O(